lms/templates/student_account/login.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="status already-authenticated-msg hidden">
     <% if (context.currentProvider) { %>
         <p class="message-copy">
-            <%- _.sprintf( gettext("You've successfully signed into %(currentProvider)s, but your %(currentProvider)s account isn't linked with an %(platformName)s account. To link your accounts, go to your %(platformName)s account settings."), context ) %>
+            <%- _.sprintf( gettext("You've successfully signed into %(currentProvider)s, but your %(currentProvider)s account isn't linked with an %(platformName)s account. To link your accounts, login now using your %(platformName)s password."), context ) %>
```

Comment:
@srpearce Would it work to say this? "You have successfully signed into %(currentProvider)s, but your %(currentProvider)s account is not linked with **any** %(platformName)s account. To link your accounts, sign in now using your %(platformName)s password."  I don't think we should say "*this* edX account" before the user has signed in to an edX account.